tensorpack/dataflow/imgaug/geometry.py

```diff
@@ -43,17 +72,21 @@ class RotationAndCropValid(ImageAugmentor):
         Note that this will produce images of different shapes.
     """
 
-    def __init__(self, max_deg, interp=cv2.INTER_LINEAR):
+    def __init__(self, max_deg, interp=cv2.INTER_LINEAR, step_deg=0):
```

Comment:
Why require max_deg==180?

For balanced probability, it's enough to require 2 * max_deg % step_deg == 0.